src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1694,8 +1694,10 @@ private void ConvertProjectReferencesToMetadataReferences_NoLock(ProjectId proje
             {
                 var referenceInfo = GetReferenceInfo_NoLock(projectIdToRetarget);
 
-                foreach (var convertedReference in referenceInfo.ConvertedProjectReferences.ToList())
+                for (var i = 0; i < referenceInfo.ConvertedProjectReferences.Count; i++)
```

Comment:
Added PerformanceSensitive attribute with explanation. Will mark resolved when pushed.

Reply:
I'll admit I'm not a huge fan of PerformanceSensitive here because:

1. This isn't something I can imagine us plausibly catching with an analyzer. If I recall one of the original goals of PerformanceSensitive was to allow tooling, but this seems a bit of a stretch.
2. The comment isn't anywhere near the offense. If this was changed you wouldn't see the attribute in the git diff.

Reply:
Updated locally; will mark resolved on push